lib/msal-node/src/client/ConfidentialClientApplication.ts

```diff
@@ -222,11 +222,11 @@ export class ConfidentialClientApplication
         const clientSecretNotEmpty = !!configuration.auth.clientSecret;
         const clientAssertionNotEmpty = !!configuration.auth.clientAssertion;
         const certificate = configuration.auth.clientCertificate || {
-            thumbprint: Constants.EMPTY_STRING,
+            thumbprintSha2: Constants.EMPTY_STRING,
             privateKey: Constants.EMPTY_STRING,
         };
         const certificateNotEmpty =
-            !!certificate.thumbprint || !!certificate.privateKey;
```

Comment:
I think both properties need to be checked since the use of "thumbprint" is still allowed (deprecated, not broken!)